core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-code/dot-container-code.component.scss

```diff
@@ -97,3 +105,10 @@
         margin-right: $spacing-2;
     }
 }
+
+.addTabIcon {
```

Comment:
`css` is case sensitive, we should use `kebab case`  instead of `camel case`. Useful read [ css naming conventions that will save you hours of debugging ](https://www.freecodecamp.org/news/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849/)